solr/solr-ref-guide/src/replication-screen.adoc

```diff
@@ -16,10 +16,10 @@
 // specific language governing permissions and limitations
 // under the License.
 
-The Replication screen shows you the current replication state for the core you have specified. <<solrcloud.adoc#solrcloud,SolrCloud>> has supplanted much of this functionality, but if you are still using Master-Slave index replication, you can use this screen to:
+The Replication screen shows you the current replication state for the core you have specified. <<solrcloud.adoc#solrcloud,SolrCloud>> has supplanted much of this functionality, but if you are still using Primary-Secondary index replication, you can use this screen to:
 
 . View the replicatable index state. (on a master node)
```

Comment:
missed?

Reply:
Missed what here?

Reply:
"View the replicatable index state. (on a master node)"